hack/make/.go-autogen

```diff
@@ -1,13 +1,12 @@
 #!/usr/bin/env bash
 
 LDFLAGS="${LDFLAGS} \
-	-X \"github.com/docker/docker/dockerversion.Version=${VERSION}\" \
-	-X \"github.com/docker/docker/dockerversion.GitCommit=${GITCOMMIT}\" \
-	-X \"github.com/docker/docker/dockerversion.BuildTime=${BUILDTIME}\" \
-	-X \"github.com/docker/docker/dockerversion.PlatformName=${PLATFORM}\" \
-	-X \"github.com/docker/docker/dockerversion.ProductName=${PRODUCT}\" \
-	-X \"github.com/docker/docker/dockerversion.DefaultProductLicense=${DEFAULT_PRODUCT_LICENSE}\" \
-"
+-X \"github.com/docker/docker/dockerversion.Version=${VERSION}\" \
+-X \"github.com/docker/docker/dockerversion.GitCommit=${GITCOMMIT}\" \
+-X \"github.com/docker/docker/dockerversion.BuildTime=${BUILDTIME}\" \
+-X \"github.com/docker/docker/dockerversion.PlatformName=${PLATFORM}\" \
+-X \"github.com/docker/docker/dockerversion.ProductName=${PRODUCT}\" \
+-X \"github.com/docker/docker/dockerversion.DefaultProductLicense=${DEFAULT_PRODUCT_LICENSE}\" "
```

Comment:
Could be worth keeping the last quote on a separate line (it's easy to miss that it's there); probably that was one of the reasons to use the indentation here.